src/VecSim/vec_sim_common.h

```diff
@@ -191,7 +191,6 @@ typedef int (*timeoutCallbackFunction)(void *ctx);
 typedef enum {
     VecSim_QueryResult_OK = VecSim_OK,
     VecSim_QueryResult_TimedOut,
-    VecSim_QueryResult_Err,
```

Comment:
why this one got removed?

Reply:
It was only used for generic return from the try-catch in the wrapper